spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -688,8 +761,13 @@ reporting its` progress using the `@Status` annotation:
     @Status
     @Path("/status")
     @GET
-    public Response status(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
-        if (isFinished(lraId))  // Business logic, not provided by spec or Implementation
+    public Response status(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId,
+                           @HeaderParam(LRA_HTTP_PARENT_CONTEXT_HEADER) URI parentLRA) {
+        if (parentLRA != null) {
+            // code which is sensitive to executing with a nested context goes here
```

Comment:
just suggestion, it seems to me that this if statements may bring an impression that the user should do something with the perentLRA. Maybe we can change the comment to"
```suggestion
            // lraId is nested, the code which is sensitive to executing with a nested context goes here
```
or something similar.

Reply:
I put something similar. In fact I reworked the whole example to make it more comprehensive.